app/models/vm_scan.rb

```diff
@@ -48,20 +48,25 @@ def call_check_policy
         :server_guid => MiqServer.my_guid
       }
       inputs = {:vm => vm, :host => vm.host}
-      MiqEvent.raise_evm_job_event(vm, {:type => "scan", :suffix => "start"}, inputs, :miq_callback => cb)
+      result = MiqEvent.raise_evm_job_event(vm, {:type => "scan", :suffix => "start"}, inputs, :miq_callback => cb)
+      if result.nil?
```

Comment:
The `result` variable is not needed here.
`if !MiqEvent.raise_evm_job_event(vm, {:type => "scan", :suffix => "start"}, inputs, :miq_callback => cb)`